app/app.html

```diff
@@ -0,0 +1,17 @@
+<ion-menu id="leftMenu" [content]="content">
```

Comment:
As I mentioned before, not only did we use the `@App` decorator to declare the component as the root component, we also supplied a template for that component, which is what will actually be rendered out onto the screen.